src/Fields/abstract_field.jl

```diff
@@ -63,6 +63,7 @@ const Abstract4DField = AbstractField{<:Any, <:Any, <:Any, <:Any, <:Any, 4}
 # when topo=Bounded, and loc=Face
 @inline axis(::Colon, N) = Base.OneTo(N)
 @inline axis(index::UnitRange, N) = index
+@inline axis(index::Base.OneTo, N) = axis(UnitRange(index), N)
```

Comment:
wouldn't this work?
```suggestion
@inline axis(index::Base.OneTo, N) = index
```

Reply:
yeah, I just went through all UnitRange dispatches and added this.